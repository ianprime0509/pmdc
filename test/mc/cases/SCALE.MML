; Output: SCALE.M
; Console: SCALE.OUT

#Title Scale
#Option /v/c

; nm alg fbl
@000 006 007
; ar  dr  sr  rr  sl  tl  ks  ml  dt ams
 000 000 000 000 000 000 000 000 000 000
 000 000 000 000 000 000 000 000 000 000
 031 021 012 015 001 000 000 002 000 000
 031 021 012 015 001 000 000 002 000 000

A @0 o4l4 cdefgab>c
