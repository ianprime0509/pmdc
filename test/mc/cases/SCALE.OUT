 .MML file --> .M file Compiler ver 4.8s
		Programmed by M.Kajihara(KAJA) 2023/09/23

Part A	Length : 192
Compile Completed.
